spec/models/post_spec.rb

```diff
@@ -1,5 +1,16 @@
 require 'rails_helper'
 
 RSpec.describe Post, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+  first_user = User.create(name: 'Tom', photo: 'https://unsplash.com/photos/F_-0BxGuVvo', bio: 'Teacher from Mexico.')
+  subject { Post.create(author: first_user, title: 'Hello', text: 'This is my first post') }
+
+  before { subject.save }
+
```

Comment:
- [ ] Please add specs for validation as required, it looks something like this :point_down: 

```ruby
describe 'validations' do
    describe 'title' do
      context 'when valid' do
        it { expect(post).to be_valid }
      end
 end
```